packages/common/src/featureServiceHelpers.ts

```diff
@@ -20,29 +20,25 @@
  * @module featureServiceHelpers
  */
 
-// ------------------------------------------------------------------------------------------------------------------ //
-
 export {
   queryFeatures as rest_queryFeatures,
   addFeatures as rest_addFeatures
 } from "@esri/arcgis-rest-feature-layer";
 
 //#region Imports -------------------------------------------------------------------------------------------------------//
 
-/*import {
-  INumberValuePair,
-  IStringValuePair,
-  IItemTemplate,
-  IDependency,
-  IUpdate,
-  IPostProcessArgs
-} from "@esri/solution-common/src/interfaces";*/
-
 import * as auth from "@esri/arcgis-rest-auth";
-import * as interfaces from "./interfaces";
```

Comment:
@MikeTschudi do you have a general rule of thumb or preference when we use one pattern over the other? I had started to switch more to this as I had thought more of your stuff was using this approach.

Reply:
@jmhauck, I have been trying to use `import * as x from y` to avoid name clashes and to simplify going back to the source item--I know where to look. I was having trouble importing and exporting with the interface symbols, so I fell back to this form. With a fresh node_modules build, I see that it is not a problem, so I will PR undoing this change. Thanks for asking.

Reply:
@MikeTschudi ah gotcha